app/main.py

```diff
@@ -2,5 +2,13 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cashe_dict = {}
```

Comment:
There is a typo in the variable name 'cashe_dict'. It should be 'cache_dict' to match the function name and standard spelling.